File: Java/baseballteam/src/Dto/Pitcher.java
package Dto;

public class Pitcher {
	private int win;
	private int lose;
	private double defense;
	
	
	public Pitcher(int win, int lose, double defense) {
		super();
		this.win = win;
		this.lose = lose;
		this.defense = defense;
	}


	public int getWin() {
		return win;
	}


	public void setWin(int win) {
		this.win = win;
	}


	public int getLose() {
		return lose;
	}


	public void setLose(int lose) {
		this.lose = lose;
	}


	public double getDefense() {
		return defense;
	}


	public void setDefense(double defense) {
		this.defense = defense;
	}


	@Override
	public String toString() {
		return "Pitcher [win=" + win + ", lose=" + lose + ", defense=" + defense + "]";
	}
}

File: kotlin/test2/src/main/kotlin/dto/Pitcher.kt
package dto

class Pitcher :Human{
    var win:Int = 0
    var lose:Int = 0
    var defense:Double = 0.0

    constructor() : super(0,"",0,0.0)

    constructor(number: Int, name: String, age: Int, height: Double) : super(number, name, age, height){
        this.win = win
        this.lose = lose
        this.defense = defense
    }

    constructor(number: Int?, name: String?, age: Int?, height: Double?, win: Int?, lose: Int?, defence: Double?)


    override fun toString(): String {
        return "$win-$lose-$defense"
    }


}